.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I maintain a global .gitignore. Adding one to the repo should be a different discussion.